src/ui/components/DotComponent.tsx

```diff
@@ -23,16 +24,41 @@ const DEFAULT_SIZE = 8
 type DotComponentProps = SwiperProps & {
   index: number
   activeIndex: number
+  numberOfSteps: number
   isActive: boolean
   onPress?: () => void
 }
 
+const WHITESPACE = '\u00a0'
+
+const STATUS = {
+  IN_PROGRESS: WHITESPACE + t`en cours`,
+  DONE: WHITESPACE + t`réalisée`,
+  TODO: WHITESPACE + t`à faire`,
+}
+
 export const DotComponent: FunctionComponent<DotComponentProps> = (props) => {
   const { borderColor, fillColor } = getDotColors(props.index, props.activeIndex)
+  let status = STATUS.TODO
+  if (props.isActive) {
+    status = STATUS.IN_PROGRESS
+  } else if (props.index < props.activeIndex) {
+    status = STATUS.DONE
+  }
+  const ariaLabel =
+    t`Étape` +
+    WHITESPACE +
+    (props.index + 1) +
+    WHITESPACE +
+    t`sur` +
+    WHITESPACE +
+    props.numberOfSteps +
+    status
   return (
     <TouchableWithoutFeedback onPress={props.onPress} testID="button">
       <DotContainer>
         <Dot
+          aria-label={ariaLabel}
```

Comment:
```suggestion
          aria-label={t`Étape ${step} sur ${totalSteps} ${status}`}
```

en définissant ça au dessus:
```tsx
const STATUS = {
  IN_PROGRESS: t`en cours`,
  DONE: t`réalisée`,
  TODO: t`à faire`,
}

  const step = props.index + 1
  const totalSteps = props.numberOfSteps
```

En gros je pense qu'il y ai besoin de NBSP dans les messages aria label ?

Sinon je sais pas si l'enum `STATUS` est nécessaire.

![image](https://user-images.githubusercontent.com/10118284/148552653-44013158-9c5f-41cc-ae53-a258c8a7c335.png)